graphql/resolve/query_rewriter.go

```diff
@@ -1211,7 +1212,9 @@ func buildAggregateFields(
 				}
 				aggregateChild := &gql.GraphQuery{
 					Alias: aggregateField.DgraphAlias() + "_" + f.DgraphAlias(),
-					Attr:  strings.ToLower(function) + "(val(" + "" + f.DgraphAlias() + "_" + constructedForField + "Var))",
+					Attr: strings.ToLower(
+						function,
```

Comment:
That's an odd reformat, I think I'd prefer:

```golang
				aggregateChild := &gql.GraphQuery{
					Alias: aggregateField.DgraphAlias() + "_" + f.DgraphAlias(),
					Attr: strings.ToLower(function) + 
                                             "(val(" + "" + f.DgraphAlias() + "_" + constructedForField + "Var))",
				}

```

Reply:
sorry for the re-ask .. could you check this one https://github.com/dgraph-io/dgraph/pull/8191 (I have closed #8189 & opened #8190)